service-api/module/Application/src/Helpers/CaseOutcomeCalculator.php

```diff
@@ -33,4 +33,21 @@ public function updateSendIdentityCheck(CaseData $caseData, string $time): void
             "outcome" => $caseData->identityCheckPassed ? 'success' : 'failure',
         ]);
     }
+
+    public function updateFinaliseIdentityCheck(CaseData $caseData, string $time): void
+    {
+        // add to logs
+        $this->logger->info("Update for CaseId " .
+            $caseData->id . "- Result: " . json_encode($caseData->caseAssistance));
+
+        $caseData->caseAssistance &&
+        $this->eventSender->send("identity-check-assistance", [
+            "reference" => "opg:" . $caseData->id,
+            "actorType" => $caseData->personType,
+            "lpaIds" => $caseData->lpas,
+            "time" => $time,
+            "assistance" => $caseData->caseAssistance->assistance,
+            "assistance_details" => $caseData->caseAssistance->details
+        ]);
```

Comment:
I don't think we're ready to send this information to Sirius yet, because we haven't had a conversation about what information they need or what they're expected to do with it.

Reply:
I suggest we leave the new function in CaseOutcomeCalculator but don't call it for the time being?